docs/process/integrating-with-fxa.md

```diff
@@ -4,28 +4,123 @@ title: Integration with FxA
+1. Type of integration (web site vs native app vs extension vs in browser)
+2. Do you know how to implement OAuth?
+3. Will you need access to a [user’s profile data][profile-data]?
+   - If yes, will you need to write to a user’s profile data?
+4. Will you need to access Sync data?
+5. Will you need encryption keys to encrypt user data?
+6. Will you need ongoing access to either a user’s profile or Sync data?
+7. Will your application display its own “enter your email” form?
+8. Who are the stakeholders?
+9. Who can be contacted for important updates, e.g., API changes?
+10. What are the QA dates?
+11. What is the target release date?
+12. Is your integration going to provide a subscription service?
+  - If yes, what products will your integration provide service for?
+13. Is your integration going to include subscription lead pages?
 
-### Sample Timeline
+The meeting request should be sent to fxa-staff[at]mozilla.com. Please be prepared to discuss these integration topics with the FxA team.
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chickweed potato bell pepper artichoke.
+## OAuth Integration
 
-## For Product Managers
+### Development
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chick
+1. A basic understanding of [OAuth 2.0][oauth] is required.
+2. Register for OAuth credentials at https://oauth-stable.dev.lcip.org/console/login. See [OAuth credentials](#oauth-credentials).
+3. Development servers point to: https://stable.dev.lcip.org
+4. User authentication follows the [OAuth 2.0][oauth] protocol.
+5. If the multiple Service Providers (SPs) are accessed, distinct tokens are created with appropriate scopes for each SP.
```

Comment:
It's not entirely clear to me what this means. I think this is talking about the case where you're talking to two different API servers, e.g. in the case of secure-proxy which was talking to both the profile-server and a third-party API service, and we wanted to make sure it used separate oauth access tokens for each?

If so, a couple of suggestions:
* I dont think "Service Provider" is standard terminology; it's used in SAML to mean something like RP, but I think the OAuth terminology we want here is [resource server](https://www.oauth.com/oauth2-servers/the-resource-server/).
* This probably belongs in a lower-numbered bullet point, after all the bits about how to execute the flow successfully, because it's something the RP will do after the oauth flow is complete.
* We probably need to say a few more words, along the lines of "If talking to multiple Resource Servers, create a distinct access token for communicating with each server, limited to only the scopes required by that server".